main.py

```diff
@@ -126,9 +126,15 @@ async def on_message(self, message) -> None:
 
                 if message.channel.id not in command_function.command_data["allowed_channels"] \
                         and message.channel.id not in configuration.ALLOWED_COMMAND_CHANNELS:
-
+                    
+                util.choose_random
+               
                     await message.channel.send(f"Please use <#{configuration.DEFAULT_COMMAND_CHANNEL}> for bot commands!")
                     return
+                or
+                    await message.channel.send(f"Use <#{configuration.DEFAULT_COMMAND_CHANNEL}> for that. I don't want to repeat myself, so please do it in right channel")
```

Comment:
Having this occur at 50% of the time might be a bit much. Also, easter eggs add complexity and make the code harder to maintain and read.
Might be beneficial to have a configurable pool of comments, instead of if statements